src/utils/__tests__/array_find_index.test.ts

```diff
@@ -1,27 +1,12 @@
-/**
- * Copyright 2015 CANAL+ Group
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+import { describe, beforeEach, afterEach, it, expect, vi } from "vitest";
+import arrayFindIndex from "../array_find_index";
```

Comment:
I'm not sure if the copyright header needs to be on every file, but it seems it has been removed by mistake ?

Reply:
No I originally added them because it was there in other files, kind of out of respect for my elders basically :p

Yet we already have a global LICENSE file and those notices clutter the files too much, so now that we technically refactor those files, I profit from it by removing those - silently enough as not to wake up any potential over-zealous lawyer :shushing_face: